ui/app/pages/swaps/actionable-message/actionable-message.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import InfoTooltip from '../../../components/ui/info-tooltip';
 
 export default function ActionableMessage({
   message = '',
   primaryAction = null,
   secondaryAction = null,
   className = '',
+  infoTooltipText = '',
 }) {
   return (
     <div className={classnames('actionable-message', className)}>
+      {infoTooltipText && (
+        <div className="actionable-message__info-tooltip-wrapper">
```

Comment:
This wrapper div isn't necessary. The `InfoTooltip` class already includes a wrapper div, and you can add classnames to it via the `wrapperClassName` prop.

Reply:
Thanks. wrapperClassName used in 9db4b4925